climpred/bootstrap.py

```diff
@@ -51,6 +51,44 @@ def _resample(hind, resample_dim):
     return smp_hind
 
 
+def _resample_idx(init, bootstrap, dim='member', replace=True):
+    """Resample bootstrap times over dim."""
+
+    def xr_broadcast(x, idx):
+        return np.moveaxis(x.squeeze()[idx.squeeze().transpose()], 0, -1)
+
+    # I get errors if I have a 0 in dim
+    if 0 in init[dim].values:
```

Comment:
You can drop `.values` here so this isn't loaded into memory. `.data` would keep it out of memory, but this works with just `if 0 in init[dim]`.